src/PointwiseFunctions/GeneralRelativity/SpacetimeNormalVector.hpp

```diff
@@ -27,9 +27,9 @@ namespace gr {
 /*!
  * \ingroup GeneralRelativityGroup
  * \brief  Computes spacetime normal vector from lapse and shift.
- * \details If \f$N, N^i\f$ are the lapse and shift respectively, then
- * \f{align} n^t &= 1/N \\
- * n^i &= -\frac{N^i}{N} \f}
+ * \details If \f$\alpha, \beta^i\f$ are the lapse and shift respectively, then
+ * \f{align} n^t &= 1/\alpha \\
```

Comment:
Could you add blank lines around the equation here like you've done elsewhere? And also one between the `\brief` and `\details`?